FireboltDotNetSdk.Tests/Integration/ConnectionTest.cs

```diff
@@ -14,32 +14,34 @@ internal class ConnectionTest : IntegrationTest
         [TestCase(true, true, Description = "Connect with both engine and database")]
         public void ConnectTest(bool useDatabase, bool useEngine)
         {
-            var connString = $"clientid={ClientId};clientsecret={ClientSecret};account={Account};env={Env}";
-            string expectedDatabase = string.Empty;
-            if (useDatabase)
+            IList<string> withoutParams = new List<string>();
+            if (!useDatabase)
             {
-                connString += $";database={Database}";
-                expectedDatabase = Database;
+                withoutParams.Add(nameof(Database));
```

Comment:
This is indeed list of parameters that should be _ignored_ when creating connection string. To avoid confusion I have renamed this variable and added comment.